python/vmaf/__init__.py

```diff
@@ -291,14 +291,16 @@ def call_vmafexec(reference, distorted, width, height, pixel_format, bitdepth,
 
         if vif_enhn_gain_limit is not None:
             # FIXME: hacky - since we do not know which feature is the one used in the model,
-            # we have to set the parameter for both, which doubles the computation.
+            # we have to set the parameter for all three, at the expense of extra computation.
             vmafexec_cmd += f' --feature float_vif=vif_enhn_gain_limit={vif_enhn_gain_limit}'
             vmafexec_cmd += f' --feature vif=vif_enhn_gain_limit={vif_enhn_gain_limit}'
+            vmafexec_cmd += f' --feature vif_neg=vif_enhn_gain_limit={vif_enhn_gain_limit}'
```

Comment:
Do we need a new feature `vif_neg`? `vif_neg` is essentially `vif` with `vif_enhn_gain_limit=1.0`, isn't it?